org.eclipse.jdt.core.tests.model/src/org/eclipse/jdt/core/tests/dom/ASTConverterTest2.java

```diff
@@ -4070,7 +4010,8 @@ public void test0534() throws JavaModelException {
 		assertNotNull("No node", node);
 		assertTrue("not a field declaration", node.getNodeType() == ASTNode.FIELD_DECLARATION);
 		FieldDeclaration declaration = (FieldDeclaration) node;
-		ASTParser parser = ASTParser.newParser(AST.JLS2);
+		@SuppressWarnings("deprecation")
+		ASTParser parser = ASTParser.newParser(AST.JLS8);
```

Comment:
I wonder if we should use `AST.getAllSupportedVersions().get(0)` instead of `AST.JLS8`?

Reply:
Done. In a whole lot of other places. There is a lot to be desired in how jls is set in tests which I intend to do as follow up commits to not make this one huge.